minio/policy.py

```diff
@@ -119,6 +131,10 @@ def _new_statements(policy, bucket_name, prefix=''):
             _new_object_statement(policy, bucket_name, prefix))
 
 
+def resource_startswith(prefix, resources):
+    return list(filter(lambda r: r.startswith(prefix), resources))
```

Comment:
1. filter() always returns list by default. wrapping filter() return is unnecessary.
2. rename resource_startswith() to _filter_resources() and add a comment what this function does